modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/MessagingEngine.java

```diff
@@ -158,14 +150,6 @@ public void initialise(MessageStore messageStore,
                                           .build();
         asyncStoreTasksScheduler = Executors.newScheduledThreadPool(threadPoolCount , namedThreadFactory);
 
-        // This task will periodically flush message count value to the store
-        messageCountFlusher = new MessageCountFlusher(messageStore, messageCountFlushNumberGap);
-
```

Comment:
Even though we got rid of further development on Cassandra stores we haven't removed it right? So Cassandra stores depends on this message counter to give message counts for queues. Removing this will result in Cassandra store message counts returning incorrect message counts. That's the impact of removing this class.